keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/MemoryCloudDriverCache.kt

```diff
@@ -60,7 +60,7 @@ class MemoryCloudDriverCache(
     .expireAfterWrite(1, HOURS)
     .build<String, Subnet>()
 
-  private fun credentialBy(name: String): Credential =
+  override fun credentialBy(name: String): Credential =
```

Comment:
Making this a public part of the `CloudDriverCache` interface to assist with determining the locations configured with an account.